src/main/java/frc/robot/subsystems/shooter/ShooterConstants.java

```diff
@@ -0,0 +1,14 @@
+package frc.robot.subsystems.shooter;
+
+public class ShooterConstants {
+  public static final int k_leftShootMotorID = 17;
+  public static final int k_rightShootMotorID = 16;
+  public static final int k_smartCurrentLimit = 40;
+  public static final double k_shootSpeaker = 1.0;
+  public static final double k_shootAmp = 0.40;
+  public static final double k_shootZero = 0.0;
+  public static final double k_shootReverseSlow = 0.30;
+  public static final boolean k_isInverted = true;
+
+  public static final double k_waitTime = 1.0;
```

Comment:
Need a more descriptive name than just "waitTime"

Reply:
We actually don't need this variable anymore, so it's been deleted.